templates/components-custom/ec-component-card/card-thumbnail.html.twig

```diff
@@ -0,0 +1,150 @@
+{#
+  /**
+   * @file
+   * Theme implementation for a ECL thumbnail list item component.
+   *
+   * @see @ecl/card.html.twig
```

Comment:
We should use the "Content item" composition instead of the card component: https://ec.europa.eu/component-library/ec/templates/compositions/content-item/code/

The composition was not available at the time of writing, it looks more like it is today on ECL 1.x:

![image](https://user-images.githubusercontent.com/153362/62631040-5e1c6a00-b930-11e9-97c5-1e893e149055.png)

Reply:
Changed.
new template `compositions/ec-component-content-item/content-item.html.twig`